tests/test_cmph.py

```
@@ -97,6 +103,34 @@ def test_filename_usage(tmpdir):
             assert mph(word) == mph2(word)
 
 
+def _entropy(strs):
+    p, lns = Counter(strs), float(len(strs))
+    return -sum(x / lns * math.log(x) for x in p.values())
+
+
+@given([unicrud_type], Random)
+def test_unicode_input(unicrud, rng):
+    unicrud = list(set(unicrud))
+    assume(len(unicrud) > 10)
+
+    # MPH is an entropy game, hence things with low-entropy will
+    # confuse the hash algorithms preventing convergence on a
+    # solution, making this test fail
+    assume(_entropy(unicrud) == -0.0)
```

why `-0.0`? is there a way to ensure enough entropy in general for the tests to pass?

assuming not, this seems like a sensible approach to intentionally failing the tests based on entropy instead of having it seem like the logic is broken.